native/explorer/Cargo.toml

```diff
@@ -86,7 +83,7 @@ features = [
 ]
 
 [dependencies.polars-ops]
-version = "0.37"
+version = "0.38"
 features = ["abs", "ewma", "cum_agg", "cov"]
 
 [features]
```

Comment:
Something is happening with floats:

```elixir
 test "dividing a series with a float scalar value on the right-hand side" do
      s1 = Series.from_list([1, 2, 3])

      s2 = Series.divide(s1, -2.5)

      assert s2.dtype == {:f, 64}
      assert Series.to_list(s2) == [-0.4, -0.8, -1.2]
    end
```
```
 27) test divide/2 dividing a series with a float scalar value on the right-hand side (Explorer.SeriesTest)
     test/explorer/series_test.exs:2027
     Assertion with == failed
     code:  assert Series.to_list(s2) == [-0.4, -0.8, -1.2]
     left:  [-0.4, -0.8, -1.2000000000000002]
     right: [-0.4, -0.8, -1.2]
     stacktrace:
       test/explorer/series_test.exs:2033: (test)
```

Reply:
@lkarthee I consider this to be "normal". Some of the upgrades did change the precision. Since it's hard to keep floats precision the same, I think this is fine. We can probably round here and call it a day.